QUICKSTART.md

```diff
@@ -12,14 +12,89 @@ Once you have a cluster, you can skip to the last step, [Launch!](#launch)
 ## Dependencies
 
 EKS has a command-line tool called `eksctl` that makes cluster setup easy.
-First, get [eksctl set up](https://github.com/weaveworks/eksctl).
+Versions of eksctl starting with 0.15.0-rc.2 support Bottlerocket natively.
```

Comment:
nit: the prior line uses the code type face, so let's keep using that.

```suggestion
Versions of `eksctl` starting with `0.15.0-rc.2` support Bottlerocket natively.
```

Reply:
The code style is there to introduce a new name, but afterward it would seem like scare quotes.